src/main/java/de/tum/cit/aet/artemis/iris/service/pyris/PyrisConnectorService.java

```diff
@@ -195,4 +198,65 @@ private String tryExtractErrorMessage(HttpStatusCodeException ex) {
             return "";
         }
     }
+
+    /**
+     * Executes a webhook and send faqs to the webhook with the given variant
+     *
+     * @param executionDTO The DTO sent as a body for the execution
+     */
+    public void executeFaqAdditionWebhook(PyrisFaqWebhookDTO toUpdateFaq, PyrisWebhookFaqIngestionExecutionDTO executionDTO) {
+        var endpoint = "/api/v1/webhooks/faqs";
+        try {
+            restTemplate.postForEntity(pyrisUrl + endpoint, objectMapper.valueToTree(executionDTO), Void.class);
+        }
+        catch (HttpStatusCodeException e) {
+            log.error("Failed to send faq {} to Pyris: {}", toUpdateFaq.faqId(), e.getMessage());
+            throw toIrisException(e);
+        }
+        catch (RestClientException | IllegalArgumentException e) {
+            log.error("Failed to send faq {} to Pyris: {}", toUpdateFaq.faqId(), e.getMessage());
+            throw new PyrisConnectorException("Could not fetch response from Pyris");
+        }
+    }
+
+    /**
+     * Executes a webhook and send faqs to the webhook with the given variant
```

Comment:
comment is the same as `executeFaqAdditionWebhook`. maybe you could be more specific here (e.g. "executeFaqAdditionWebhook adds faqs...", "executeFaqDeletionWebhook deletes faqs...").